notebooks/noj_book/sklearn_reference.clj

```diff
@@ -10,11 +10,17 @@
    [noj-book.utils.render-tools-sklearn]
    [scicloj.sklearn-clj.ml]))
 
+;;## Sklearn model reference
 
+;;Below we find all sklearn models with their parameters and the original documentation.
+;;
+;;The parameters are given as Clojure keys in kebap-case. As the document texts are 
+;;imported from python they refer to the python spelling of the parameter. 
+;;
+;;But the translation between the two should be obvious.
```

Comment:
Also, I suggest the initial paragraphs will be above the `ns` definition, so that the chapter can begin in a clearer way.